scripts/zalenium.sh

```diff
@@ -354,6 +356,8 @@ StartUp()
     fi
     export ZALENIUM_DESIRED_CONTAINERS=${DESIRED_CONTAINERS}
     export ZALENIUM_MAX_DOCKER_SELENIUM_CONTAINERS=${MAX_DOCKER_SELENIUM_CONTAINERS}
+    export ZALENIUM_SELENIUM_CONTAINER_CPU_LIMIT=${SELENIUM_CONTAINER_CPU_LIMIT}
```

Comment:
I don't think it is necessary to change this file, the idea would be just to have the cpuLimit and memoryLimit as environment variables and use them when they are available.

Please revert the changes to this file and just grab the env vars in the Java code.

Reply:
if ı don't use in this sh files. When I run that script "sudo docker run --rm -ti --name zalenium -d -p 4444:4444 -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/zalenium:/home/seluser/videos --privileged zalenium:3.13.0c-SNAPSHOT start --cpuLimit 20000000 --memoryLimit 536870912",  it wouldn't work.

Reply:
It is possible to pass them as environment variables, e.g.:

```
zalenium -d -p 4444:4444 -v /var/run/docker.sock:/var/run/docker.sock 
-v /tmp/zalenium:/home/seluser/videos 
-e ZALENIUM_SELENIUM_CONTAINER_CPU_LIMIT=20000000
-e ZALENIUM_SELENIUM_CONTAINER_MEMORY_LIMIT=536870912
--privileged zalenium:3.13.0c-SNAPSHOT start ",
```
So there is no need to touch the zalenium.sh script.